database/neo4j/neo4j.test.js

```diff
@@ -0,0 +1,10 @@
+let { closeNeo4j } = require('./neo4j')
+
+describe('Neo4j database', () => {
+	closeNeo4j = jest.fn().mockReturnValue(Promise.resolve())
+	let driver.close = jest.fn().mockReturnValue(1)
+  it('it should call end when closing neo4j', async () => {
+    await closeNeo4j()
+    expect(closeNeo4j).toHaveBeenCalledTimes(1)
```

Comment:
I think expecting `closeNeo4j` to have been called is weird.

In `neo4js.js`, when you run `closeNeo4j` function, it runs `driver.close` function. and `driver` object is the return of `neo4j.driver()`. So why don't you mock `neo4j-driver` instead of `closeNeo4j`?